core/include/drivers/ns16550.h

```diff
@@ -0,0 +1,174 @@
+/*
+ * NS16550 Serial Port
+ * originally from linux source (arch/powerpc/boot/ns16550.h)
+ *
+ * Cleanup and unification
+ * (C) 2009 by Detlev Zundel, DENX Software Engineering GmbH
+ *
+ * modified slightly to
+ * have addresses as offsets from CONFIG_SYS_ISA_BASE
+ * added a few more definitions
+ * added prototypes for ns16550.c
+ * reduced no of com ports to 2
+ * modifications (c) Rob Taylor, Flying Pig Systems. 2000.
+ *
+ * added support for port on 64-bit bus
+ * by Richard Danter (richard.danter@windriver.com), (C) 2005 Wind River Systems
+ */
+
+/*
+ * Note that the following macro magic uses the fact that the compiler
+ * will not allocate storage for arrays of size 0
+ */
+#define CONFIG_SYS_NS16550_REG_SIZE 1
```

Comment:
This should be called `CFG_*` for consistency with other OP-TEE configuration variables (and same for the other ones below). And it should be defined in `core/arch/arm/plat-ls/conf.mk`, otherwise it makes no sense to keep the #ifdefs.